olp-cpp-sdk-core/CMakeLists.txt

```diff
@@ -428,6 +436,17 @@ if(OLP_SDK_ENABLE_DEFAULT_CACHE)
     )
 endif()
 
+if(OLD_SDK_ENABLE_DEFAULT_CACHE_LMDB)
+    target_include_directories(${PROJECT_NAME}
+        PUBLIC
```

Comment:
Why do we need to specify the include dir as public? As this will only be used internally it can be private as well.

Reply:
Done